docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile:  docker-files/Dockerfile-app
+    ports:
+      - "5000:5000"
+    depends_on: 
+      - "db"
+
+    environment: 
+      DB_HOST_NAME : db
+      DB_USER_NAME : root
+      DB_PASSWORD : 12345
+      DB_INSTANCE : devopsloft
+
+  db:
```

Comment:
I strongly suggest checking using pre-baked mysql image and also add a docker volume for the data